install-instructions.md

```diff
@@ -100,6 +100,13 @@ You need [Java 17 or higher](https://adoptium.net/) to play the game.
 
 </details>
 
+### [GDLauncher](https://gdlauncher.com/)
+
+1. Click `Modpacks`
+2. Select `Modrinth` tab from the left
```

Comment:
todo: discuss with launcher devs to see if MR or CF is better (ads, creator rewards, download count, updatability etc)

Reply:
This should be fixed within the latest commit (https://github.com/Fabulously-Optimized/wiki/pull/109/commits/7133e69756bd59b992c4d6ca1f8528e4cf9c9a6b). I have changed the instructions to use CurseForge instead as discussed as it gives revenue to mod / modpack creators.